lib/gooddata/lcm/user_bricks_helper.rb

```diff
@@ -0,0 +1,27 @@
+module GoodData
+  module LCM2
+    class UserBricksHelper
+      class << self
+        def resolve_client_id(domain, project, data_product)
+          md = project.metadata
+          goodot_id = md['GOODOT_CUSTOM_PROJECT_ID'].to_s
+
+          client = domain.clients(:all, data_product).find do |c|
+            c.project_uri == project.uri
+          end
+          if goodot_id.empty? && client.nil?
```

Comment:
Refers to 'goodot_id' more than self (maybe move it to another class?) (FeatureEnvy)